config/package/tmpfiles.d/uwsgi.conf

```diff
@@ -0,0 +1 @@
+d /run/uwsgi/socorro 0755 socorro nginx -
```

Comment:
When the RPM is built via [`scripts/install.sh`](https://github.com/mozilla/socorro/blob/master/scripts/install.sh#L21-L22) this conf file will get dropped into `/etc/tmpfiles.d/uwsgi.conf`. Unfortunately, according to the [packaging guidelines](https://fedoraproject.org/wiki/Packaging:Guidelines#binfmt.d.2C_sysctl.d_and_tmpfiles.d), this is an inappropriate behaviour - instead, said conf should be dropped into `/usr/lib/tmpfiles.d/`.

Interestingly, this would be easier to deal with if we were using a real spec.  In any case, we'll need to modify the behaviour of `scripts/install.sh` if we want to adhere to the guidelines.